res/themes/unthemed/client/client_light.css

```
@@ -0,0 +1,742 @@
+    border: 1px solid;
+    padding-top: 15px;
+}
+
+/* Used for Ranked Buttons only at the moment*/
+QToolButton#rankedPlay
+{
+	border-radius: 5px;
+	border: 1px solid;
+}
+
+QToolButton#rankedPlay::disabled
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: disabled

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)